client/packages/core/src/instaml.js

```diff
@@ -186,7 +186,8 @@ function expandUnlink(attrs, [etype, eidA, obj]) {
   return withIdAttrForLookup(attrs, etype, eidA, retractTriples);
 }
 
-function expandUpdate(attrs, [etype, eid, obj]) {
+function expandUpdate(attrs, [etype, eid, _obj]) {
+  const obj = immutableRemoveUndefined(_obj);
```

Comment:
```suggestion
function expandUpdate(attrs, [etype, eid, obj_]) {
  const obj = immutableRemoveUndefined(obj_);
```

`_obj` makes me think that the variable isn't going to be used.